cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/preview/DefaultPreviewRunner.java

```diff
@@ -117,7 +125,8 @@
                        ProgramNotificationSubscriberService programNotificationSubscriberService,
                        LevelDBTableService levelDBTableService,
                        StructuredTableAdmin structuredTableAdmin,
-                       StructuredTableRegistry structuredTableRegistry) {
+                       StructuredTableRegistry structuredTableRegistry, PreviewRunWrapper previewRunWrapper,
```

Comment:
Having a class name with `Wrapper` at the top level is awkward. Typically wrapper class should be private/internal. Why we need this interface? In standalone we can write out the same file and delete it, isn't it?

Reply:
Thanks for the explanation! Updated code to remove the `PreviewRunWrapper` interface since it was not used in Standalone. 
Added `DistributedPreviewRunner` which extends from the `DefaultPreviewRunner` and does the job of file handling.
Please check again.